spec/features/users/regular_users/show_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.describe "User Profile" do
+  before :each do
+    @regular_user = User.create!(name: "George Jungle",
+                  address: "1 Jungle Way",
+                  city: "Jungleopolis",
+                  state: "FL",
+                  zipcode: "77652",
+                  email: "junglegeorge@email.com",
+                  password: "Tree123")
+
+    allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(@regular_user)
```

Comment:
I'm not great with stubs so maybe I'm just being dense, but I don't see how this is necessary for the test to pass. Why do we need a stub if we are creating an actual regular user?

Reply:
so basically, in the user#show method, if we need the ``@user =User.find(session[:user_id])`` cannot find the session[:user_id]